extensions/amp-image-slider/0.1/amp-image-slider.js

```diff
@@ -620,10 +661,10 @@ export class AmpImageSlider extends AMP.BaseElement {
     leftPercentage = this.limitPercentage_(leftPercentage);
 
     this.updateTranslateX_(this.bar_, leftPercentage);
```

Comment:
It's not super clear what `leftPercentage` means here. Is it % from left? % left-shifted? Why are you adjusting the right image with a "left" percentage? Could we fix the variable naming here so this is clear? =)